### ISSUE TITLE
Card – Test Accessibility with Bill Tipton

### ISSUE THREAD

**Issue by taysea** [grommet/hpe-design-system#2911]
The recent card work should be validated with Bill Tipton to ensure our considerations re: accessibility are truly creating an intuitive experience for screen reader users.

Task screen:  https://erot25.csb.app/
Code: https://codesandbox.io/s/card-accessibility-testing-v1-erot25?file=/index.js
Script (WIP): https://hpe-my.sharepoint.com/:w:/p/vicky_avalos/EcviJ8fcFDBOvYM1RwnCv7MBBB7yLkuKuZ5K-zm4g6f9uA?email=taylor.seamans%40hpe.com&e=1PeK1N

- [x] Create script (2hr)
- [x] Receive feedback/review
- [x] Implement feedback + finalize script for testing (30min-1 hr)
- [x] Test accessibility tasks with Bill (1 hr)

Success criteria Matt has suggested we utilize to reference back to and evaluate whether the script touches on the points we want to test and learn.

For example: A screen reader user should be able to:

- [x] identify what subject a collection of cards represent
- [x] how many items are part of that collection
- [x] easily traverse each item in the collection and find any of interest
- [x] quickly decide whether a card may or may not be something they are interested in
- [x] get more detail on items which they may potentially be interested in
- [x] know how to get to the full detail behind the card’s content

Follow-on work to be filed depending on results of tests:
- [ ] Make modifications to card implementation and guidance (length TBD)

**Comment by vavalos5** [comment]
Second round of script has been created. As a next step, I reached out to Maria if she has the time to review it and let me know if she has any suggestions on phrasing questions or feedback.

**Comment by vavalos5** [comment]
Also asked @taysea @halocline and @SOjaHPE to review. This can be done early next week. Please know that testing is scheduled for Wednesday afternoon at 1pm PDT.

**Comment by vavalos5** [comment]
I've received feedback from Maria, Scott and Taylor 🥳

**Comment by vavalos5** [comment]
Feedback has been reviewed and implemented.

**Comment by vavalos5** [comment]
Testing with Bill is complete. Here is the link to the recording: https://hpe-my.sharepoint.com/:v:/p/vicky_avalos/ERDY7t8xO8pNvufdFlbLTcEBAuIr-5nreAU_i4KTELPGSQ

Notes link: coming soon!

**Comment by taysea** [comment]
Follow-on work is filed here https://github.com/grommet/hpe-design-system/issues/2929
